src/components/FilePath.tsx

```diff
@@ -38,7 +37,12 @@ export const FilePath: React.FunctionComponent<IFilePathProps> = (
 
   return (
     <React.Fragment>
-      <span className={getFileIconClass(props)} />
+      <LabIcon.resolveReact
```

Comment:
This is a nice use of `resolveReact`. Its use is appropriate here since either/both of `filetype.icon` and `filetype.iconClass` may be undefined